portal/views/portal.py

```diff
@@ -1051,7 +1051,7 @@ def celery_test(x=16, y=16):
     context = {"id": res.task_id, "x": x, "y": y}
     result = "add((x){}, (y){})".format(context['x'], context['y'])
     task_id = "{}".format(context['id'])
-    result_url = url_for('.celery_result', task_id=task_id)
+    result_url = url_for('portal.celery_result', task_id=task_id)
```

Comment:
Needed to explicitly reference the full path since celery_test is called from a different blueprint.